### PR TITLE
Docker-updates

### PR DESCRIPTION
**Pull request by justineechen**
Removed spotify docker maven plugin, packageFile as tar.gz in pom.xml and updated Dockerfile to use ADD instruction

Addressing OpenLiberty/guides-common#299

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -122,6 +122,14 @@ To build the application, run the Maven `package` goal from the command line in
 mvn clean package
 ```
 
+Run the following commands to build and containerize the application:
+
+[role='command']
+```
+docker build -t system:1.0-SNAPSHOT system/.
```
Comment (evelinec) [inline README.adoc]: There're 4 services, system, inventory, gateway and job, plus 2 other services, kafka and zookeeper. I guess all need to be built with the new `docker build` command? @proubatsis what do you think?

**Comment by evelinec** [comment]
@justineechen can you test the changes in all 3 platforms, Windows, Linux and Mac?

File: README.adoc (hunk 1 above)
Comment (proubatsis) [reply README.adoc]: Yes, they all need to be built.

File: finish/inventory/pom.xml [hunk 2]
```diff
@@ -151,15 +151,11 @@
                     <execution>
                         <id>package-server</id>
                         <configuration>
-                            <packageFile>${project.build.directory}/defaultServer.zip</packageFile>
```
Comment (proubatsis) [inline finish/inventory/pom.xml]: These pom changes need to be done for `job` and `gateway` as well to stay consistent with these changes. @justineechen @evelinec

**Comment by evelinec** [comment]
@proubatsis @gkwan-ibm can you take a quick look? Thanks.

**Comment by gkwan-ibm** [comment]
@justineechen 
the gateway and job pom.xml need to remove 
 ```
           <plugin>
                <groupId>com.spotify</groupId>
                <artifactId>dockerfile-maven-plugin</artifactId>
           </plugin>
```

**Pull request merged**